apis/apps/v1alpha1/imagepulljob_types.go

```diff
@@ -83,6 +83,11 @@ type ImagePullJobNodeSelector struct {
 
 // PullPolicy defines the policy of the pulling task
 type PullPolicy struct {
+	// Specifies the type of Pod ImagePullPolicy of the pulling task.
+	// Defaults to IfNotPresent
+	// +optional
+	Type PullPolicyType `json:"type,omitempty"`
```

Comment:
How about just reusing `PullPolicy` of  package "k8s.io/api/core/v1".

Reply:
That's what I did at first, but Never seemed to make no sense in this scenario, so I tried defining a type in the same way as CompletionPolicyType